docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,381 @@
+* The server already has a properly configured switch with 802.1X support.
+* The PacketFence RADIUS server is working in your environment.
+* You have a Microsoft Windows 2008 R2 Entreprise server installed.
+* In this guide you will see a lot of use of <ServerDNSName>, most of the MSPKI services requires in their configuration to use the FQDN of the server and not his IP.
+* The PacketFence management IP will be 192.168.1.5.
+* The RADIUS shared secret is "useStrongerSecret".
+
+Installation
+------------
+
+Step 1: Install Active Directory Certificate Service (ADCS)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+NOTE: This will cover the configuration for Active Directory Certificate Services (ADCS) on Microsoft Windows 2008 R2 Entreprise. This configuration has been tested to be integreted with PacketFence. The installation of ADCS is not cover by this guide see the Microsoft white paper about it for more information (http://social.technet.microsoft.com/wiki/contents/articles/9063.network-device-enrollment-service-ndes-in-active-directory-certificate-services-ad-cs.aspx).
+
+For the integration with PacketFence the following subroles need to be installed in ADCS:
+
+* Certification Authorithy Web Enrollment
+* Network Device Enrollment Service
+* Online Responder
+
+Before start the configuration, a hotfix is necessary due to a Microsft issue. After restarting the service ADCS, the server cannot enroll new certificate with the error message: "The RPC Server is unavailable". The hotfix is available here: https://support.microsoft.com/en-us/kb/2633200
+
+Communication between MSPKI and PacketFence will be using port 80 with this configuration.
+
+Configuring Network Device Enrollement Service (NDES)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+NOTE: In this guide you will see a lot of use of <ServerDNSName>, most of the MSPKI services requires in their configuration to use the FQDN of the server and not his IP.
+
+Our main activity in the deployment of ADCS is to configure Network Enrollement Device Service (NDES) which will allow us to exchange certificate with the MS PKI server via Simple Certificate Exchange Protocol (SCEP).
+
+Every configuration change has to be done by an account with administrative privileges.
+
+To allow the use of NDES as a Pki-Provider in BOYD deployement there is multiple changes.
+
+Challenge Password
+^^^^^^^^^^^^^^^^^^
+
+MSCEP include by default a challenge password, which is unique and dynamically generated for each device which want to enroll on the system. The issue here while using the challenge password is that it is necessary for an administrator to connect to the NDES server from the user's device. This a barrier to BOYD deployement for instance, where you want the user to be able to register himself his device. Since we use NDES with PacketFence, our security to obtain a certificate would be the credentials necessary to access the enrollement system.
```

Comment:
Replace `This a barrier to BOYD deployement for instance, where you want the user to be able to register himself his device.` by the following

In a BYOD deployment, this can be a barrier as a user cannot register a device by himself without the intervention of an administrator